plugins/org.jboss.tools.jst.js.npm/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
I would prefer to not include .settings to the individual projects. Let's use the workspace settings instead. We might want to use a project specific settings in some cases. But this is not the case.